src/screens/SettingsScreen.js

```diff
@@ -42,14 +42,14 @@ const onDeactivatePushNotifications = (revert) => {
   setInAppPermission(false).then((success) => !success && revert());
 };
 
-const Filter = {
+const FILTER = {
```

Comment:
following the changes, this should be also capitalized entirely?

```
const FILTER = {
  GENERAL: 'general',
  LIST_TYPES: 'listTypes'
};
```

as well as renamed to `LIST_FILTER` as we have now the separation across code with `TOP_FILTER` and this one here?

Reply:
✅